README.md

```diff
@@ -1,2 +1,58 @@
-# AwesomeBooks-Module
-Microverse project for practicing implementing modules (javascript)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+# AwesomeBooks-Project
+Awesome books: plain JavaScript with objects
+
+## Acces link to live version
+
+> https://vitorguedesmadeira.github.io/AwesomeBooks-Module/
+
+## Getting Started
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+- A modern web browser like Chrome, Safari, Firefox, etc.
+- JavaScript enabled.
+
+Setup
+- Open the Repository Link of this project in GitHub.
+
+Install
+- Clone the repository on your local machine, using this command:
+git clone https://github.com/VitorGuedesMadeira/AwesomeBooks-Module.git
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- LocalStorage
+- VScode
+- JS Modules
+
+## Author
+
+👤 **Vitor Guedes Madeira**
+
+- GitHub: [@vitorguedesmadeira](https://github.com/VitorGuedesMadeira)
+- Twitter: [@CDahguedes](https://twitter.com/CDahguedes)
+- LinkedIn: (https://www.linkedin.com/in/v%C3%ADtor-guedes-madeira-1691aa15a/)
```

Comment:
- [optional] It is best practice to use your username Wich will point to the link provided as you did it on GitHub and twitter. You can do the same on LinkedIn.